I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 468, in script
    $ renpy.display_menu(actions.items())
  File "game/script.rpy", line 468, in <module>
    $ renpy.display_menu(actions.items())
NameError: name 'actions' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 468, in script
    $ renpy.display_menu(actions.items())
  File "D:\renpy\renpy-8.2.0-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\renpy\renpy-8.2.0-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 468, in <module>
    $ renpy.display_menu(actions.items())
NameError: name 'actions' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.2.0.24012702
virgin 1.0
Sat May 18 13:33:57 2024
